### DISCUSSION TITLE
Programatically submit the form

### DISCUSSION THREAD

**Discussion by kilitary** [livewire/livewire#7997] [1. Help] [answered]
Livewire version
3.4.6

Which PHP version are you using?
PHP 8.3.3

Repository URL
No response

Steps To Reproduce
No response

Additional information

Would like to use custom time-triggered/UI-involved submitting the form. But in docs i only see input type=submit. I tryed using hooks and alerts and did not find the solution.

I have this:
```

	<form wire:submit id='form_{{$prop->wire_hash()}}'>
			<input type='hidden' value='{{$prop->id}}'>

			<div wire:dirty.class="buzy" wire:key="{{ $prop->wire_hash() }}" class='property-el propertie-grid-item'>
                <span>
                    <label for='name'>название
	                <div>
		                <input value='{{$prop->name}}'>
	                </div></label>
                </span>
				<label for='name'>тип</label>
				<div>
					<select>
						@foreach ($types as $type)
							<option
								wire:key="{{$type->wire_hash()}}"
								value='{{$type->id}}'
								@if($type->id == $prop->type_id) selected @endif>
								{{$type->name}}
							</option>
						@endforeach
					</select>
				</div>
				<span>
                    <button wire:click="delete({{$prop->id}})" wire:confirm="точно ебнуть?">удалить</button>
                </span>
			</div>

		</form>
```

Also:

`<select wire:keydown="what_to_put_here_tosubmit_the_form">` inside form.

**Comment by shubdragon** [comment, marked as answer]
if you have dynamic form you could do something like this

`
    <form wire:submit id='form_{{$prop->wire_hash()}}' x-ref="formBlah">
         <select x-on:keydown="$refs.formBlah.submit()">
    </form>
`

**Reply by kilitary** [reply]
thanks +shubdragon
